src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -58,64 +112,55 @@ public ArmSubsystem(int armId, String armCanbus, int pizzaBoxId, String pizzaBox
-      }      
       for (int i = 0; i < 5; ++i) {
         motorStatus = m_pizzaBox.getConfigurator().apply(configs);
         if (motorStatus .isOK()) break;
       }
       if(!motorStatus.isOK()) {
         System.out.println("Could not apply configs, error code: " + motorStatus.toString());
       }
-      if(!motorStatusArm.isOK()) {
-        System.out.println("Could not apply configs, error code: " + motorStatusArm.toString());
-      }
-  
-      m_encoder.reset();
-      m_encoder.setSamplesToAverage(5);
-      m_encoder.setDistancePerPulse(1. / 256.);
-      m_encoder.setMinRate(1.0);
-    
-  }
-
-  // Sets arm angle in degrees with given velocity and acceleration
-  public void setAngle(double angle, double vel, double accel) {
-
-    if(angle < Constants.Arm.ARM_MIN_ANGLE) { //Will not be less than minimum angle
-      angle = Constants.Arm.ARM_MIN_ANGLE;
-    }
-    else if (angle > Constants.Arm.ARM_MAX_ANGLE) { // Will not be greater than maximum angle
-      angle = Constants.Arm.ARM_MAX_ANGLE;
-    }
-
-    double adjustedAngle = (((angle - encoderGetAngle() + getArmAngle())) * Constants.Arm.GEAR_RATIO)/Constants.Arm.ROTATION_TO_DEGREES;
-    MotionMagicVoltage m_smoothArmMovement = new MotionMagicVoltage(adjustedAngle, false, 0, 0, false, false, false);
 
-    var talonFXConfigs = new TalonFXConfiguration();
-    talonFXConfigs.Slot0.kS = .24;
-    talonFXConfigs.Slot0.kV = .12;
-    talonFXConfigs.Slot0.kP = 4.8;
-    talonFXConfigs.Slot0.kI = 0;
-    talonFXConfigs.Slot0.kD = .1;
+    Shuffleboard.getTab("Arm").addDouble("Encoder Angle", ()->encoderGetAngle()).withWidget(BuiltInWidgets.kGraph)
+    .withSize(3,3)
+    .withPosition(0, 0);
+    Shuffleboard.getTab("Arm").addDouble("Motor Angle", ()->getArmAngle()).withWidget(BuiltInWidgets.kGraph)
+    .withSize(3,3)
+    .withPosition(3, 0);
 
+    Shuffleboard.getTab("Arm").add("Arm PID", this.getController());
+  }
 
-    var motionMagicConfigs = talonFXConfigs.MotionMagic;
-    motionMagicConfigs.MotionMagicCruiseVelocity = vel;
-    motionMagicConfigs.MotionMagicAcceleration = accel; 
-    motionMagicConfigs.MotionMagicJerk = 1600; // 1600 rps/s^2 jerk (0.1 seconds)
-    m_arm.getConfigurator().apply(talonFXConfigs, 0.03);
-
-    m_smoothArmMovement.Slot = 0;
+  //Looking at the left of the robot, counterclockwise arm spin is positive
+  // Sets arm angle in degrees with given velocity and acceleration
+  //VelocityVoltage​(double Velocity, double Acceleration, boolean EnableFOC, double FeedForward, int Slot, boolean OverrideBrakeDurNeutral, boolean LimitForwardMotion, boolean LimitReverseMotion)
+ public void spinArm(double speed)
+ {
+  if(speed < -1) { //Will not be less than minimum angle
+    speed = -1;
+  }
+  else if (speed > 1) { // Will not be greater than maximum angle
+    speed = 1;
+  }
+  m_arm.set(-speed);
+ }
 
-    m_arm.setControl(m_smoothArmMovement);
+ public void targetArmAngle(double angle)
+ {
+  double calculatedAng = angle - Arm.ENCODER_OFFSET;
```

Comment:
Arm min angle is the minimum angle the arm can go to (45 degrees); this is around the angle needed for intake.